README.md

```diff
@@ -109,6 +109,8 @@ target is a url (http load tests) or host:port (grpc health test).  flags are:
 	Path to the certificate file to be used for GRPC server TLS
   -compression
 	Enable http compression
+  -content-type
```

Comment:
I noticed in my other PR that the flags section of the README is actually slightly out of sync, from a white space point of view, with the source, did you run the full 

go install .
release/updateFlags.sh

and copy paste the whole section into the readme ?

Reply:
I did